container_test.go

```diff
@@ -1091,10 +1093,7 @@ func TestAttachToContainerRawTerminalFalse(t *testing.T) {
 		Stream:       true,
 		RawTerminal:  false,
 	}
-	err := client.AttachToContainer(opts)
-	if err != nil {
-		t.Fatal(err)
-	}
+	client.AttachToContainer(opts)
```

Comment:
Ah no it wasn't, this was a new error because now we check the io.Copy, I missed that this was not backwards...working on it now, my mistake.